src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023imaginginterface.py

```diff
@@ -0,0 +1,123 @@
+from dateutil.parser import parse as dateparse
+import datetime
+import json
+import glob
+
+from abdeladim_2023imagingextractor import (
+    Abdeladim2023MultiPlaneImagingExtractor,
+    Abdeladim2023SinglePlaneImagingExtractor,
+)
+from neuroconv.datainterfaces.ophys.baseimagingextractorinterface import BaseImagingExtractorInterface
+from neuroconv.utils import FolderPathType
+from neuroconv.utils.dict import DeepDict
+from roiextractors.extractors.tiffimagingextractors.scanimagetiff_utils import extract_extra_metadata
+from roiextractors.extractors.tiffimagingextractors.scanimagetiffimagingextractor import ScanImageTiffSinglePlaneImagingExtractor
+
+class Abdeladim2023SinglePlaneImagingInterface(BaseImagingExtractorInterface):
+    """
+    Data Interface for writing imaging data for the MouseV1 to NWB file using Abdeladim2023SinglePlaneImagingExtractor.
+    """
+
+    Extractor = Abdeladim2023SinglePlaneImagingExtractor
+
+    @classmethod
+    def get_available_channels(cls, folder_path):
+        file_path = sorted(glob.glob(f"{folder_path}/*.tif"))
```

Comment:
`natsort.natsorted` is better to make sure the file names are in the correct order

Reply:
I will change this in #5

Reply:
I will change this in #5